### DISCUSSION TITLE
How to change location of images from buildah bud from /var/tmp?

### DISCUSSION THREAD

**Discussion by jadewilk** [containers/buildah#3671] [Q&A]
Exactly as the title says. I have a small root disk, and then a large secondary disk, meaning that buildah does not have enough space to put the container into tmp, so I'd like to change where it does this.
`
error copying layers and metadata for container "5a533d94e23eb5a3d1996812c6e84adcc33a7dcd367923b3b843b6553b65ac66": initializing source containers-storage:09a503effcb4bc2ca0c269e7ecd82261cb5781f440fdcc95bfabe0b851052444-working-container: error storing layer "b682a3bb4bcf788eca56701f7c83debf63cc93a0ef60de27876a1a9f6f7c853e" to file: write /var/tmp/buildah744289050/layer: no space left on device
`
I'm thinking of something similar to how with docker, where I put an option like this in the daemon.json config to change where buildkit stores its stuff.
`"data-root": "/data/docker/"
`

**Comment by flouthoc** [comment]
Hi @wiryfuture You need to set `image_copy_tmp_dir` as specified here https://github.com/containers/common/blob/main/docs/containers.conf.5.md

**Reply by jadewilk** [reply]
Where did I make a mistake?
![Screenshot_20211215_190159](https://user-images.githubusercontent.com/50517631/146248708-1bb69767-aefa-48fc-88b2-a8bd2c54fdcd.png)

**Reply by rhatdan** [reply]
Most likely not supported in the version of the engine you are using.

**Comment by rhatdan** [comment]
You can change storage.conf to place the storage in a different location. Similarly you can specify TMPDIR and buildah will store large temporary files there.
